ipwb/replay.py

```diff
@@ -179,6 +179,126 @@ def showMementosForURIRs_sansJS():
     return redirect('/memento/*/' + urir, code=301)
 
 
+def lookup_keys(surt):
+    keyre = re.compile(r"(.+)([,/]).+")
+    key = surt.split("?")[0].strip("/")
+    keys = [key, f"{key}/*"]
+    while "," in key:
+        try:
+            m = keyre.match(key)
+            keys.append(f"{m[1]}{m[2]}*")
+            key = m[1]
+        except Exception as e:
+            key = key.strip("/,")
+
+    return keys
+
+
```

Comment:
This does not belong here either? This was specific to MementoMap where I had to create all partial key for lookup.

```suggestion
```